runtime/service_test.go

```diff
@@ -170,21 +170,29 @@ func TestBuildVMConfiguration(t *testing.T) {
 
 	for _, tc := range testcases {
 		tc := tc // see https://github.com/kyoh86/scopelint/issues/4
+		tempDir, err := ioutil.TempDir(os.TempDir(), namespace)
```

Comment:
If you move this inside the anonymous function inside `t.Run`, you can do this:

```go
tempDir, err := ioutil.TempDir(os.TempDir(), namespace)
require.NoError(t, err)
defer os.RemoveAll(tempDir)
```

That will give you cleanup per sub-test and keep the directory scoped to each subtest.